controllers/profiles/common/app_properties.go

```diff
@@ -156,7 +194,7 @@ func (a *appPropertyHandler) withDataIndexServiceUrl() AppPropertyHandler {
 	if profiles.IsProdProfile(a.workflow) && dataIndexEnabled(a.platform) {
 		a.addDefaultMutableProperty(
 			dataIndexServiceUrlProperty,
-			fmt.Sprintf("%s://%s.%s/processes", dataIndexServiceUrlProtocol, GetDataIndexName(a.platform), a.platform.Namespace),
+			fmt.Sprintf("%s://%s.%s/processes", dataIndexServiceUrlProtocol, GetServiceName(a.platform.Name, DataIndexService), a.platform.Namespace),
```

Comment:
does a job service app property also need to be added here so that workflows know how to communicate to it?

Reply:
Looks like it [has](https://kiegroup.github.io/kogito-docs/serverlessworkflow/latest/job-services/quarkus-extensions.html#kogito-addons-quarkus-jobs-knative-eventing) at least 3 from what I can gather by the data index property. 
```
mp.messaging.outgoing.kogito-job-service-job-request-events.connector=quarkus-http
mp.messaging.outgoing.kogito-job-service-job-request-events.url=${K_SINK:http://localhost:8280/v2/jobs/events}
mp.messaging.outgoing.kogito-job-service-job-request-events.method=POST
```
But, as it [happens](https://github.com/apache/incubator-kie-kogito-serverless-operator/blob/162f3693844de04e94b447b29253eb3ed38057ad/controllers/profiles/common/app_properties.go#L44) with data index, only the `URL` property is added in the code for data index, the following being the equivalent for job service:
```
mp.messaging.outgoing.kogito-job-service-job-request-events.url
```

I will follow example from data index and add this last property if the job service is enabled